stdlib/asm/crypto/stark/constants.masm

```diff
@@ -300,6 +327,62 @@ export.number_queries_ptr
     push.NUM_QUERIES_PTR
 end
 
+export.set_number_queries
+    push.NUM_QUERIES_PTR mem_store
+end
+
+export.get_number_queries
+    push.NUM_QUERIES_PTR mem_load
+end
+
+export.remainder_poly_size_ptr
+    push.REMAINDER_POLY_SIZE_PTR
+end
+
+export.set_remainder_poly_size
+    push.REMAINDER_POLY_SIZE_PTR mem_store
+end
+
+export.get_remainder_poly_size
+    push.REMAINDER_POLY_SIZE_PTR mem_load
+end
+
+export.num_fri_layers_ptr
+    push.NUM_FRI_LAYERS_PTR
+end
+
+export.set_num_fri_layers
+    push.NUM_FRI_LAYERS_PTR mem_store
+end
+
+export.get_num_fri_layers
+    push.NUM_FRI_LAYERS_PTR mem_load
+end
+
+export.remainder_poly_address_ptr
+    push.REMAINDER_POLY_ADDRESS_PTR
+end
+
+export.set_remainder_poly_address
+    push.REMAINDER_POLY_ADDRESS_PTR mem_store
+end
+
+export.get_remainder_poly_address
+    push.REMAINDER_POLY_ADDRESS_PTR mem_load
+end
+
+export.fri_queries_address_ptr
+    push.FRI_QUERIES_ADDRESS_PTR
+end
```

Comment:
I might have missed it, but are we using this procedure anywhere?

Reply:
Removed this and other unused procedures